src/py21cmfast/wrapper.py

```diff
@@ -2875,6 +2899,10 @@ def run_lightcone(
                     )
                 lc_index += n
                 box_index += n
+                rot_index += n
+                if rotation_cubes and rot_index >= user_params.HII_DIM:
+                    los_axis = (los_axis + 1) % 3
+                    rot_index -= user_params.HII_DIM
```

Comment:
why is rot_index not used in the code? I can't really remember what was doing before

Reply:
I think you've asked the same question before. rot_index is checked (rot_index >= user_params.HII_DIM) to change the LOS since I'm changing LOS after every HII_DIM.